composer.json

```diff
@@ -21,7 +21,8 @@
         "symfony-cmf/testing": "^1.3@dev",
         "symfony/dependency-injection": "^2.0.5|3.*",
         "symfony/config": "^2.2|3.*",
-        "symfony/event-dispatcher": "^2.1|3.*"
+        "symfony/event-dispatcher": "^2.1|3.*",
+        "friendsofsymfony/jsrouting-bundle": "@dev"
```

Comment:
Testing requires it. `@dev` isn't needed, only to support Symfony 3 as a Symfony 3 supporting version has not been released.

However, as minimum stability is now set to stable for all CMF repositories, we need to explicitely allow FOSJsRoutingBundle to be installed in dev when needed.